core/server/services/members/index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    get api() {
+        return require('./api');
+    }
+};
```

Comment:
Thinking out loud, I am bit skeptical if this is the best structure for root of members service, and to expose APIs as this directly. In my head I feel service should expose a function, like `init`, so we can pass in deps in future, starting with member models. Ignore if thats what you had in mind and this is just for first version 😬

Reply:
If we look at the auth service, the files, e.g. `api.js` in this case would pass in the deps, and require the models etc... I think either way works, but we should add it when we need it